Classes/Mvc/Request.php

```diff
@@ -33,12 +33,12 @@
 class Request
 {
     /**
-     * @var array
+     * @var mixed[]
```

Comment:
what is the benefit of this notation? Imho there is none. And it makes it harder to grasp the type